modules/core/src/main/java/org/apache/ignite/configuration/ClientConnectorConfiguration.java

```diff
@@ -349,6 +357,31 @@ public ClientConnectorConfiguration setSelectorCount(int selectorCnt) {
         return this;
     }
 
+    /**
+     * Gets maximum allowed number of active connections per node.
```

Comment:
```suggestion
     * Gets maximum allowed number of active client connections per node.
```

Reply:
Is this shared with JDBC thin or not?

Reply:
Yes, shared with ODBC and JDBC. Clarified in comment.